webpack.standalone.config.js

```diff
@@ -36,6 +36,9 @@ switch ( env ) {
 		config.devtool = 'source-map';
 		config.devServer = {
 			contentBase: path.resolve( __dirname, 'targets/standalone' ),
+			historyApiFallback: {
+				index: 'example.html',
```

Comment:
I think the example HTML and assets are missing?

Reply:
Forgot to push them. It should be ok now